### PR TITLE
Profile edit page

### PR DESCRIPTION
**Pull request by tahminehrivandi**
please check my profile-edit form

### CODE REVIEW

**Comment by AmirHosseinKarimi** [comment]
screenshot

Fit fields width like as grid on design. 
<details><summary>Browser metadata</summary>





```
Path:      /src/pages/profile-edit.html
Browser:   Firefox 116.0 on Linux x86_64
Viewport:  2560 x 1283 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-20--simple-shop-technullogy.netlify.app%2Fsrc%2Fpages%2Fprofile-edit.html&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=116.0&resolution=2560x1283&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-20--simple-shop-technullogy.netlify.app/src/pages/profile-edit.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6494788e3056180008f42116&commentID=64bd2c2bc1e59423fd3b7447&resolution=resolved)

**Comment by AmirHosseinKarimi** [comment]
Use `<input type="test" inputMode="numeric" />` for numeric inputs with numeric keyboard without increase/decrease buttons.

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /src/pages/profile-edit.html
Browser:   Firefox 116.0 on Linux x86_64
Viewport:  2560 x 1283 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-20--simple-shop-technullogy.netlify.app%2Fsrc%2Fpages%2Fprofile-edit.html&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=116.0&resolution=2560x1283&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-20--simple-shop-technullogy.netlify.app/src/pages/profile-edit.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6494788e3056180008f42116&commentID=64bd2e18b5c95225857632be&resolution=resolved)

**Comment by AmirHosseinKarimi** [comment]
Its a [select](https://www.w3schools.com/tags/tag_select.asp) element like as drop-down. Not a text input.

screenshot


<details><summary>Browser metadata</summary>





```
Path:      /src/pages/profile-edit.html
Browser:   Firefox 116.0 on Linux x86_64
Viewport:  2560 x 1283 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-20--simple-shop-technullogy.netlify.app%2Fsrc%2Fpages%2Fprofile-edit.html&os=OS%20X&os_version=Big%20Sur&browser=Firefox&browser_version=116.0&resolution=2560x1283&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-20--simple-shop-technullogy.netlify.app/src/pages/profile-edit.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=6494788e3056180008f42116&commentID=64bd2ea2efc13a25528a5c33&resolution=resolved)

File: src/pages/profile-edit.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    {{> header}}
+    <div class="flex justify-center bg-[#F5F5F5]">
+   <div class="m-10 flex flex-col gap-y-6 rounded  p-3 " dir="rtl">
+    <form class="flex flex-col gap-3  bg-[#FFFFFF] h-[528px] w-[986px] ">
+      <h2 class="w-[249px] h-[36px]  not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
+
+      <div class="grid grid-cols-3 gap-x-6 ">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16.8px] text-[#7E8AAB] mr-5 mt-5">نام و نام خانوادگی</label>
+          <input type="text" class="w-[230px] h-[50px] bg-[#F7F9FC] text-right mr-5 pr-3" value="علیض">
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:20]:
```suggestion
          <input type="text" class="w-[230px] h-[50px] bg-[#F7F9FC] text-right mr-5 pr-3" value="" placeholder="علیرضا">
```

Comment (AmirHosseinKarimi) [reply src/pages/profile-edit.html:20]: Clear all field values & add an example value as a placeholder.

File: src/pages/profile-edit.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Profile Edit</title>
+ 
+  </head>
+  <body>
+    {{> header}}
+    <div class="flex justify-center bg-[#F5F5F5]">
+   <div class="m-10 flex flex-col gap-y-6 rounded  p-3 " dir="rtl">
+    <form class="flex flex-col gap-3  bg-[#FFFFFF] h-[528px] w-[986px] ">
+      <h2 class="w-[249px] h-[36px]  not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:15]: Remove all fixed widths & heights. Elements should have dynamic size. Instead, use `line-height` to set the element's inner height.

```suggestion
      <h2 class="not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
```

File: src/pages/profile-edit.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Profile Edit</title>
+ 
+  </head>
+  <body>
+    {{> header}}
+    <div class="flex justify-center bg-[#F5F5F5]">
+   <div class="m-10 flex flex-col gap-y-6 rounded  p-3 " dir="rtl">
+    <form class="flex flex-col gap-3  bg-[#FFFFFF] h-[528px] w-[986px] ">
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:14]: Use TailwindCSS classes.

```suggestion
    <form class="flex flex-col gap-3  bg-white h-[528px] w-[986px] ">
```

File: src/pages/profile-edit.html [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+      <h2 class="w-[249px] h-[36px]  not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
+
+      <div class="grid grid-cols-3 gap-x-6 ">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16.8px] text-[#7E8AAB] mr-5 mt-5">نام و نام خانوادگی</label>
+          <input type="text" class="w-[230px] h-[50px] bg-[#F7F9FC] text-right mr-5 pr-3" value="علیض">
+          <img  class="  absolute bottom-3 right-16 mr-2" src="/src/assets/images/pages/ui/crusore-icon.png" alt="crusore-image">
+        </div>
+        <div class="flex flex-col gap-2">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5">شماره موبایل</label>
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:24]: Remove all `not-italic` classes. It is useless.

File: src/pages/profile-edit.html [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        <div class="flex flex-col gap-2 relative ">
+          <label class=" w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 ">آدرس ایمیل</label>
+          <input type="email" class="  w-[230px] h-[50px] bg-[#F7F9FC] text-right pr-3 " value="efwabifawafup@gmail.com ">
+          <img  class="  absolute bottom-3.5 left-7 ml-20 " src="/src/assets/images/pages/ui/crusore-icon.png" alt="crusore-image">
+        </div>
+      </div>
+  
+      <div class="grid grid-cols-2 gap-x-6">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 mr-5">استان</label>
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:36]: Use TailwindCSS classes.

```suggestion
          <label class="w-[88px] h-[17px]  not-italic font-light text-xs text-[#7E8AAB] mt-5 mr-5">استان</label>
```

File: tailwind.config.js [hunk 6]
```diff
@@ -3,7 +3,9 @@
 export default {
   content: ["./*.html", "./src/**/*.html"],
   theme: {
-    extend: {},
+    extend: {
+      
+    },
```
Comment (AmirHosseinKarimi) [inline tailwind.config.js:8]: Useless change...
Prevent any useless changes & config your IDE according to project settings.

```suggestion
    extend: {},
```

File: src/pages/profile-edit.html [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+            <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5">آدرس</label>
+            <input type="text" class="w-[450-px] h-[56px] bg-[#F7F9FC] text-right pr-3  ml-5 mt-25 " value="ششششششششششششششششصیصش" />
+          </div>
+          </div>
+          <div class="grid grid-cols  ">
+            <div class="flex flex-col items-center relative ">
+              <button type="submit" class="  w-[930px] h-[60px] bg-[#00ADB5] absolute right-7 top-8  ">
+                <span class="w-[102px] h-[25px] mt-[17.5px] font-semibold text-[18px] leading-[25.2px] text-[#FFFFFF] text-center">ذخیره تغییرات</span>
+              </button>
+              <img class="  absolute top-12 right-10" src="/src/assets/images/pages/ui/profile-circle.png"   alt="profile-image">
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:62]: Remove any unnecessary spaces between attributes & values.
Use Prettier formater

```suggestion
              <img class="absolute top-12 right-10" src="/src/assets/images/pages/ui/profile-circle.png" alt="profile-image">
```

File: src/pages/profile-edit.html [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+      </div>
+  
+      <div class="grid grid-cols-2 gap-x-6">
+        <div class="flex flex-col gap-2 relative">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5 mr-5">استان</label>
+          <input type="text" class=" w-[454px] h-[56px] bg-[#F7F9FC] text-right pr-3  mr-5 mt-25 " placeholder="انتخاب کنید">
+          <img  class="  absolute bottom-5 right-20 mr-[350px]" src="/src/assets/images/pages/ui/Arrows-icon.png" alt="crusore-image">
+        </div>
+        <div class="flex flex-col gap-2">
+          <label class="w-[88px] h-[17px]  not-italic font-light text-[12px] leading-[16 .8px] text-[#7E8AAB] mt-5">شهر</label>
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:41]: Use the nearest TailwindCSS classes if there is a small difference.

**Review by AmirHosseinKarimi — CHANGES_REQUESTED** [review]

**Comment by tahminehrivandi** [comment]
I edit my profile edit page please check it_

File: tailwind.config.js [hunk 9]
```diff
@@ -3,7 +3,16 @@
...
   content: ["./*.html", "./src/**/*.html"],
   theme: {
-    extend: {},
+   
+
+    extend: {
+      colors:{
+        'blue': '#7E8AAB',
+        'light': '#F7F9FC',
+        'gray': '#7a7f8d',
```
Comment (AmirHosseinKarimi) [inline tailwind.config.js:12]: TailwindCSS has these colors by default. If you need to add any other color define them with a different name.
Use [this tool](https://chir.ag/projects/name-that-color) to name colors.

File: src/pages/profile-edit.html [hunk 10]
```diff
@@ -5,70 +5,76 @@
...
-          </div>
-          </div>
-          <div class="grid grid-cols  ">
-            <div class="flex flex-col items-center relative ">
-              <button type="submit" class="  w-[930px] h-[60px] bg-[#00ADB5] absolute right-7 top-8  ">
-                <span class="w-[102px] h-[25px] mt-[17.5px] font-semibold text-[18px] leading-[25.2px] text-[#FFFFFF] text-center">ذخیره تغییرات</span>
-              </button>
-              <img class="  absolute top-12 right-10" src="/src/assets/images/pages/ui/profile-circle.png"   alt="profile-image">
+   <div class="flex justify-center bg-neutral-100">
+      <div class="m-10 flex flex-col gap-y-6 rounded p-3" dir="rtl">
```
Comment (AmirHosseinKarimi) [inline src/pages/profile-edit.html:12]: Change the page direction instead of elements.

**Comment by AmirHosseinKarimi** [comment]
Hi Tahmineh. I hope you are doing well.
First of all. Good job :clap: I can see your progress and am happy for you. Keep going girl.

I refactored the code and applied some changes in a [new branch](https://github.com/TechNullogy/Simple-Shop/tree/profile-edit-page).
You can pull this branch like as main branch and create a new pull request to merge with the main branch.
Feel free to apply any changes to the code.

**Pull request closed without merging**